e2e/__tests__/__snapshots__/coverageHandlebars.test.ts.snap

```diff
@@ -0,0 +1,371 @@
+              "column": 0,
+              "line": 13,
+            },
+          },
+          Object {
+            "end": Object {
+              "column": 8,
+              "line": 13,
+            },
+            "start": Object {
+              "column": 2,
+              "line": 13,
+            },
+          },
+        ],
+        "type": "binary-expr",
+      },
+      "5": Object {
+        "loc": Object {
+          "end": Object {
+            "column": 8,
+            "line": 13,
+          },
+          "start": Object {
+            "column": 2,
+            "line": 13,
+          },
+        },
+        "locations": Array [
+          Object {
+            "end": Object {
+              "column": 8,
+              "line": 13,
+            },
+            "start": Object {
+              "column": 2,
+              "line": 13,
+            },
+          },
+          Object {
+            "end": Object {
+              "column": 8,
+              "line": 13,
+            },
+            "start": Object {
+              "column": 2,
+              "line": 13,
+            },
+          },
+        ],
+        "type": "cond-expr",
+      },
+    },
+    "f": Object {
+      "0": 1,
+      "1": 0,
+      "2": 1,
+    },
+    "fnMap": Object {
+      "0": Object {
+        "decl": Object {
+          "end": Object {
+            "column": null,
+            "line": 8,
+          },
+          "start": Object {
+            "column": 10,
+            "line": 8,
+          },
+        },
+        "loc": Object {
+          "end": Object {
+            "column": null,
+            "line": 8,
+          },
+          "start": Object {
+            "column": 10,
+            "line": 8,
+          },
+        },
+        "name": "(anonymous_0)",
+      },
+      "1": Object {
+        "decl": Object {
+          "end": Object {
+            "column": null,
+            "line": 10,
+          },
+          "start": Object {
+            "column": 8,
+            "line": 10,
+          },
+        },
+        "loc": Object {
+          "end": Object {
+            "column": null,
+            "line": 10,
+          },
+          "start": Object {
+            "column": 8,
+            "line": 10,
+          },
+        },
+        "name": "(anonymous_1)",
+      },
+      "2": Object {
+        "decl": Object {
+          "end": Object {
+            "column": null,
+            "line": 6,
+          },
+          "start": Object {
+            "column": 2,
+            "line": 6,
+          },
+        },
+        "loc": Object {
+          "end": Object {
+            "column": null,
+            "line": 6,
+          },
+          "start": Object {
+            "column": 2,
+            "line": 6,
+          },
+        },
+        "name": "(anonymous_2)",
+      },
+    },
+    "path": "greet.hbs",
+    "s": Object {
+      "0": 1,
+      "1": 0,
+      "2": 2,
+      "3": 1,
+    },
+    "statementMap": Object {
```

Comment:
This snapshot is huge. Can we instead assert on specific values plus the shape?

Reply:
it's the fact it exists at all.

However ,maybe we could snapshot the report istanbul outputs to the terminal instead of the source map itself? Like in `transform.test.ts`: https://github.com/facebook/jest/blob/fafb5b7b1598693371cc8113d253abf0981209f3/e2e/__tests__/__snapshots__/transform.test.ts.snap#L12-L45

Reply:
Definitely better

Reply:
Sure, how’s this?